ansible/roles/ocp4-workload-pam-fraudmanagement-workshop-verification/tasks/workload_per_project_amqstreams.yml

```diff
@@ -1,16 +1,18 @@
 ---
 - name: Wait until KafkaCluster has Ready condition
-  command: >
-    oc get kafka/{{ _namespace }}-cluster -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' -n "{{ _namespace }}"
+  command: >-
+    oc get kafka/{{ _namespace }}-cluster
+    -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' -n "{{ _namespace }}"
   register: kafka
   retries: "{{ _retry }}"
   delay: "{{ _delay }}"
-  until: kafka.stdout == "True"
+  until: kafka.stdout == "true"
 
 - name: Wait until KafkaTopic block-account has Ready condition
-  command: >
-    oc get KafkaTopic/block-account -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' -n "{{ _namespace }}"
+  command: >-
+    oc get KafkaTopic/block-account
+    -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' -n "{{ _namespace }}"
   register: blocktopic
   retries: "{{ _retry }}"
   delay: "{{ _delay }}"
-  until: blocktopic.stdout == "True"
\ No newline at end of file
+  until: blocktopic.stdout == "true"
```

Comment:
If this is evaluating a string, it needs to be tested before changing case like this.